packages/docs/src/components/page-configs/contribution/translation/page-config.ts

```diff
@@ -1,14 +1,15 @@
-import { ApiDocsBlock } from '../../../../types/configTypes'
-import { DocsHelper } from '../../../../helpers/DocsHelper'
+import { ApiDocsBlock } from '@/types/configTypes'
+import { DocsHelper } from '@/helpers/DocsHelper'
 import { t } from '@/helpers/I18nHelper'
+import { languages } from '@/components/languages'
+
+const translationStatuses = languages.map((lang) => [`${lang.name} (${lang.code})`, lang.status])
 
 export default [
   DocsHelper.title('translation.title'),
   DocsHelper.paragraph('translation.description'),
   DocsHelper.table(
     [t('translation.table.language'), t('translation.table.supported')],
-    [
-      ['English (en)', '+'],
-      ['Spain (es)', 'partial'],
-    ]),
+    translationStatuses
```

Comment:
I think we will need translation for "language" and "translation status" later as well, but I'll create a new issue for that.